primefaces/src/main/java/org/primefaces/util/ExcelStylesManager.java

```diff
@@ -87,14 +95,44 @@ public static ExcelStylesManager createExcelStylesManager(Workbook wb, Locale lo
      *
      * @param column the column from which value comes
      * @param cell the cell to operate on
-     * @param value the String value to put in the cell
+     * @param value the ColumnValue value to put in the cell
      */
-    public void updateCell(UIColumn column, Cell cell, String value) {
-        updateCell(cell, value);
+    public void updateCell(UIColumn column, Cell cell, ColumnValue value) {
+        if (value.isCustomValue()) {
+            updateCellCustomValue(cell, value.getCustomValue());
+        }
+        else {
+            updateCellFallbackValue(cell, value.getFallbackValue());
+        }
         applyColumnAlignments(column, cell);
```

Comment:
That's not my intended use, I want to be capable of tell PrimeFaces I'm submitting a String value so ExcelExporter could treat like that and stop trying to guess what type is when stronglyTypedCells = true. I want to handle as customValue if the value was set with `exportValue` or `exportFunction` attribute in `<p:column>` despite of what type the value is. In all other cases, behave like before (ExcelExporter tries to figure what type is from the rendered value of the `<p:column>`)

Reply:
First 2 lines will cause regression as explained before. Don't think custom/fallback have a lot of meaning here, it's more whether it's the raw value (which will require conversion) or string (where the conversion has already been made, but stronglyTypedCells  should still be taken account to remain backward compatible)

Reply:
I've already change the name of the variable to "explicitlySetValue" instead of customValue to reflect what I want to mean. For now stronglyTypedCells is considered when the user don't use `exportValue` or `exportFunction` attributes of `<p:column>`, otherwise treats the value with type of the object. Maybe I could add another <p:column> attribute which permits disabling stronglyTypedCells per column basis.

Reply:
I think that new attribute in `<p:column>` could be called `exportOptions` and it will be a new interface similar to org.primefaces.component.export.ExporterOptions but for columns instead of whole DataTable.